src/packagedcode/pypi.py

```diff
@@ -94,9 +94,9 @@ class PythonPackage(models.Package):
     extensions = ('.egg', '.whl', '.pyz', '.pex',)
     default_type = 'pypi'
     default_primary_language = 'Python'
-    default_web_baseurl = None
-    default_download_baseurl = None
-    default_api_baseurl = None
+    default_web_baseurl = 'https://pypi.org'
+    default_download_baseurl = 'https://pypi.io/packages/source'
```

Comment:
@JonoYang can you check this https://github.com/pypa/pypi-legacy/issues/438#issuecomment-226940730 that was  suggested by @Kwpolska